### PR TITLE
[ONNX] Support annotated models but run without export

### PR DESCRIPTION
**Pull request by disktnk**
The model has `annotate` (or `anchor`) but run the model no under `ppe.onnx.export`, occurs an error.

```
>       for name, child_module in _annotation_init.model.named_children():
E       AttributeError: '_AnnotationInit' object has no attribute 'model'
```

This PR fix the error. `annotate` works only when `onnx.export` mode.

### CODE REVIEW

**Review by kmaehashi — APPROVED** [review]
LGTM!

File: pytorch_pfn_extras/onnx/annotate.py [hunk 1]
```diff
@@ -191,7 +193,9 @@ def annotate(**attrs):
     Args:
         attrs (dict): annotation parameters
     """
-    return _Annotation(**attrs)
+    if torch.onnx.is_in_onnx_export():
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/annotate.py:196]: So `annotate`  and `scoped_anchor` would be ignored when it's not exporting onnx
👍

**Review by take-cheeze — APPROVED** [review]

**Comment by take-cheeze** [comment]
CI failure seems to be caused by mypy. Should we wait https://github.com/pfnet/pytorch-pfn-extras/pull/209 ?

**Comment by kmaehashi** [comment]
Sorry that log is difficult to understand. Currently mypy is enabled but errors are ignored. The true error in this PR is the flake8 issue:

```
/src/tests/pytorch_pfn_extras_tests/onnx/test_annotate.py:26:1: E303 too many blank lines (3)
```

File: tests/pytorch_pfn_extras_tests/onnx/test_annotate.py [hunk 2]
```diff
@@ -22,6 +22,30 @@
 torch_version = version.Version(torch.__version__)
 
 
+
```
Comment (kmaehashi) [inline tests/pytorch_pfn_extras_tests/onnx/test_annotate.py:25]:
```suggestion
```

**Comment by kmaehashi** [comment]
Thanks @disktnk and @take-cheeze!

**Pull request merged**